airflow/contrib/kubernetes/pod.py

```diff
@@ -114,3 +118,4 @@ def __init__(
         self.hostnetwork = hostnetwork or False
         self.tolerations = tolerations or []
         self.security_context = security_context
+        self.envs_from_configmaps = envs_from_configmaps or []
```

Comment:
The other variable is just called `secrets` - so maybe we just call this `configmaps`? I think with the doc string you already have this would be clear enough.

Reply:
@ashb fixed.